analyzer/flow_server.go

```diff
@@ -81,7 +87,23 @@ func (c *FlowServerWebSocketConn) OnMessage(client shttp.WSSpeaker, m shttp.WSMe
 		logging.GetLogger().Errorf("Error while parsing flow: %s", err.Error())
 		return
 	}
-	logging.GetLogger().Debugf("New flow from Websocket connection: %+v", f)
+	//logging.GetLogger().Debugf("New flow from Websocket connection: %+v", f)
+	if len(c.ch) >= c.maxFlowBufferSize {
+		c.numOfLostFlows++
+		if c.timeOfLastLostFlowsLog.IsZero() ||
+			(c.timeOfLastLostFlowsLog.Sub(time.Now()) >= time.Second) {
```

Comment:
time.Now().Sub(c.timeOfLastLostFlowsLog), otherwise it will never match

Reply:
Done